lib/helpers.js

```diff
@@ -19,4 +22,17 @@ module.exports = {
             .request(method, url, request, options);
     },
 
+    readLaunchesFromFile() {
+        return fs.readFileSync(tempFileForLaunchIds, 'utf8', function(err, data) {
```

Comment:
You can't pass callback for `fs.readFileSync`. you have to try-catch exceptions

Reply:
Thanks, good point! Done